app/helpers/dot_helper.rb

```diff
@@ -8,14 +8,25 @@ def render_agents_diagram(agents, layout: nil)
         } rescue false)
       decorate_svg(svg, agents).html_safe
     else
-      uriquery = URI.encode_www_form(cht: 'gv', chl: agents_dot(agents))
-      #Get query maximum length should be under 2048 bytes with including "chart?" of google chart request url
-      if uriquery.length > 2042
-        "Too many agent to display, please check unused agents"
+      # Google chart request url
+      url = URI("https://chart.googleapis.com/chart")
+      http = Net::HTTP.new(url.host, url.port)
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Is this necessary in your environment?  It is not preferable in general to disable verification.